web/src/components/home-page/intro-video.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Box, Container, SystemStyleObject } from 'theme-ui';
+import YouTubeVideo from './youtube-video';
+
+interface IntroVideoProps {
+  videoUrl?: string;
+  videoTitle?: string;
+}
+
+const style: SystemStyleObject = {
+  '.video-container': {
+    position: 'absolute',
+    top: 0,
+    bottom: 0,
+    right: 0,
+    left: 0,
+    zIndex: -99,
+    '&::after': {
+      display: 'block',
+      content: '',
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      width: '100%',
+      height: '800px',
+      zIndex: 1,
+    },
+  },
+  '.data-container': {
+    minHeight: '3rem',
+    fontSize: 5,
+    justifyContent: 'flex-start',
+    alignContent: 'center',
+    alignItems: 'center',
+    height: '800px',
+    fontWeight: 'bold',
+    display: 'flex',
+    textAlign: 'start',
+    verticalAlign: 'center',
+    color: 'background',
+    px: '188px',
```

Comment:
Can you use a dynamic value here? `188px` feels like something that will cause us problems in the future.

Reply:
You could also use the size matrix if it should be a fixed value.